package/CHANGELOG

```diff
@@ -36,6 +36,8 @@ API Changes
     style format '.npz'. Offsets files will be regenerated when you load a
     xtc/trr trajectory again. (Issue #441)
   * rotation_matrix now accepts array-likes as input
+  * rmsd doesn't superimpose by default anymore. The superposition
```

Comment:
@kain88-de please move to 0.14.1; we'll rename that release entry to 0.15.0 later.